app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,21 @@
+<%# <footer>
```

Comment:
Can we just delete this layout instead of commenting it out, or is it coming to be making a comeback soon?

Reply:
yeah, like within 4 hours